pkgs/tools/backup/zfs-replicate/default.nix

```
@@ -1,22 +1,28 @@
-{ buildPythonApplication, click, fetchPypi, hypothesis, pytest
-, lib, stringcase
+{ buildPythonApplication
+, click
+, fetchPypi
+, pytestCheckHook
+, hypothesis
+, pytest
+, lib
+, stringcase
 }:
 
 buildPythonApplication rec {
   pname = "zfs-replicate";
-  version = "1.2.3";
+  version = "3.1.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "b2cb9d4670a6e12d14a446c10d857862e91af6e4526f607e08b41bde89953bb8";
+    hash = "sha256-24816c3e92ea9abc03e1ce281b5fcce4dfb5534092096e3e2c57a0d585ec4d67";
```

ofborg eval error:
```
       error: invalid SRI hash '24816c3e92ea9abc03e1ce281b5fcce4dfb5534092096e3e2c57a0d585ec4d67'
```
---
```suggestion
    hash = "sha256-mRINo20/uFlWtP5W7w+D2E9o89hlAsqZmBjuv0qWP9k=";
```